engines/wintermute/base/base_object.cpp

```diff
@@ -102,7 +102,7 @@ BaseObject::BaseObject(BaseGame *inGame) : BaseScriptHolder(inGame) {
 	_shadowModel = nullptr;
 	_posVector = Math::Vector3d(0.0f, 0.0f, 0.0f);
 	_angle = 0.0f;
-	_scale = 1.0f;
+	_scale3D = 1.0f;
```

Comment:
Consider running something like the static-analyzer in Clang to see if there's other similar mistakes.